src/test/resources/features/DeletePaticipant.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Delete Participant
+  In order to delete the participants
+  As a admin
+  I want to delete a participant
+
+  Background:
+    Given There is a registered administrator with username "admin" and password "password" and email "admin@sample.app"
+    And There is a registered user with username "user" and password "password" and email "user@sample.app"
+
+  Scenario: Admin deletes participant
+    Given I login as "admin" with password "password"
+    And I create a participant with role "SECRETARY"
+    When I delete a participant with id "1"
+    Then The response code is 204
+    And The previously deleted participant doesn't exist
+    And The response code is 404
```

Comment:
This should be part of the previous step "The previously deleted participant does not exist". So you should check it in that step.